currency-rates/internal/app/app.go

```diff
@@ -31,11 +30,18 @@ type App struct {
 func NewApp(c Config, logger tools.Logger, ctx context.Context) (*App, error) {
 	nbuClient := clients.NewNBUClient(logger)
 	privatClient := clients.NewPrivatClient(logger)
-	rateService := services.NewRate(logger, []services.RateClient{nbuClient, privatClient})
+
+	nbuRateGetter := clients.NewClient("nbu", nbuClient)
+	privatRateGetter := clients.NewClient("privat", privatClient)
```

Comment:
For did you make this logic?
Isn't  it enough just to use nbuClient and privatClient?

Reply:
I thought all clients should be combined. But yes, you're right - it's redundant